salt/config/__init__.py

```diff
@@ -2848,14 +2848,14 @@ def is_profile_configured(opts, provider, profile_name, vm_=None):
     # If cloning on Linode, size and image are not necessary.
     # They are obtained from the to-be-cloned VM.
     if driver == 'linode' and profile_key.get('clonefrom', False):
-        non_image_drivers.append('linode')
+        drivers_requiring_an_image.append('linode')
```

Comment:
No need to rename ``non_image_drivers`` to ``drivers_requiring_an_image`` as they are totally opposite